sdks/wasm/framework/src/WebAssembly.Net.Http/WasmHttpMessageHandler.cs

```diff
@@ -168,8 +146,10 @@ private async Task doFetch (TaskCompletionSource<HttpResponseMessage> tcs, HttpR
 
 				HttpResponseMessage httpresponse = new HttpResponseMessage ((HttpStatusCode)Enum.Parse (typeof (HttpStatusCode), status.Status.ToString ()));
 
-				httpresponse.Content = StreamingSupported && StreamingEnabled
-				    ? new StreamContent (wasmHttpReadStream = new WasmHttpReadStream (status))
+				var streamingEnabled = request.Properties.TryGetValue ("StreamingEnabled", out var streamingEnabledValue) && (bool)streamingEnabledValue;
+
+				httpresponse.Content = StreamingSupported && streamingEnabled
```

Comment:
We should probably give some indication when streamingEnabled == true and StreamingSupported == false

Reply:
What would that look like? Throwing an error seems problematic since you cannot easily query `StreamingSupported` from a Blazor app,

Reply:
What happens today when you set the static `StreamingEnabled` flag to true when `StreamingSupported` is false?

Reply:
It no-ops today, without any sort of indication that it failed. It's clearly not ideal, but I'm not sure what the right way is to surface this.